src/main/java/org/gridsuite/actions/server/dto/Contingencies.java

```diff
@@ -0,0 +1,22 @@
+package org.gridsuite.actions.server.dto;
+
+import com.powsybl.contingency.Contingency;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.UUID;
+
+/*
+    @author Jamal KHEYYAD <jamal.kheyyad at rte-international.com>
+ */
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Builder
+public class Contingencies {
```

Comment:
Maybe we could find a better name for this ....
something like `ContingencyListExportResult` or something like that ?

Reply:
done